### PR TITLE
Remove Unicode quotes from help text

### PR DESCRIPTION
**Pull request by ssweeny**
Help text has Unicode quotes in a few places, which breaks help output as noted in https://bugs.launchpad.net/snapcraft/+bug/1555733. This change replaces them with ASCII quotes

### CODE REVIEW

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by kyrofa** [comment]
@snappy-m-o you're a bit over-zealous. @ssweeny ugh, thank you for tracking those down! This looks good.

**Comment by kyrofa** [comment]
I wonder if we can come up with a test that would prevent unicode characters from sneaking into the help text in the future?

**Comment by ssweeny** [comment]
Have a python script try to print out all the module help text in the source tree? :wink:

**Comment by kyrofa** [comment]
Ah, let me rephrase.

Hey @ssweeny, mind adding a test to this PR to make sure unicode characters can't sneak into the help text?

**Comment by ssweeny** [comment]
@kyrofa yeah I can give it a shot this weekend.

**Comment by sergiusens** [comment]
:+1:

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by snappy-m-o** [comment]
Can one of the admins verify this patch?

**Comment by elopio** [comment]
OK to test

**Comment by elopio** [comment]
I'm not sure why the three comments. We have to jobs, so it might make sense to have two comments. Three is jjust nonsense. Instead of two jobs we could have one job that triggers these two, we will be checking into it now that we need a workflow plugin for snappy.

**Comment by kyrofa** [comment]
Excellent job @ssweeny, thank you! Please squash up, then I'm good with this :+1: .

**Comment by kyrofa** [comment]
@elopio autopkgtest "No test results found" = "Success" ?

**Comment by ssweeny** [comment]
As best I can tell the examples test failed due to a bad download. Is there a way to re-run it?

**Comment by elopio** [comment]
retest this please

**Comment by elopio** [comment]
@kyrofa I'm not sure what to collect as results of the autopkgtest. I'll check if I can change the message.

**Comment by sergiusens** [comment]
This is good as is; landing
